src/Device.Net/ConnectedDeviceDefinitionBase.cs

```diff
@@ -36,5 +36,11 @@ public abstract class ConnectedDeviceDefinitionBase : DeviceDefinitionBase
         /// The maximum size of data to be read from the device
         /// </summary>
         public int? ReadBufferSize { get; set; }
+
```

Comment:
These cannot be added to ConnectedDeviceDefinitionBase because not all device types support these.